AISKU/Tests/testLegacySnippet.ts

```diff
@@ -0,0 +1,89 @@
+
+/**
+ * DO NOT FIX BUGS WITH THE OBJECT RETURNED BY THIS HELPER
+ * -------------------------------------------------------
+ * This is a helper that returns an object that is the same as the legacy snippet, if there is
+ * an issue with the tests because of the object this returns there is probably a bug during 
+ * initialization.
+ */
+export function createLegacySnippet(aiConfig) {
```

Comment:
This returns the full legacy snippet -- without adding to the window or attempting to drop a script tag

Reply:
It's basically a copy of the snippet with some parts commented out